toolkit/scripts/pkggen.mk

```
@@ -75,7 +75,7 @@ analyze-built-graph: $(go-graphanalytics)
 	fi
 
 # Parse all specs in $(BUILD_SPECS_DIR) and generate a specs.json file encoding all dependency information
-$(specs_file): $(chroot_worker) $(BUILD_SPECS_DIR) $(build_specs) $(build_spec_dirs) $(go-specreader)
+$(specs_file): $(chroot_worker) $(BUILD_SPECS_DIR) $(build_specs) $(build_spec_dirs) $(go-specreader) $(depend_RUN_CHECK)
```

Do we care about this if it doesn't affect the behavior of the specreader?
```suggestion
$(specs_file): $(chroot_worker) $(BUILD_SPECS_DIR) $(build_specs) $(build_spec_dirs) $(go-specreader)
```

---

Actually, probably better to continue to pass the below `--run-check` through and keep `$(depend_RUN_CHECK)`